### ISSUE TITLE
[1.6.1.1] Android target: No android targets (SDKs) installed!

### ISSUE THREAD

**Issue by chip** [meteor/meteor#9947]
Hi, I recently upgraded to 1.6.1.1 on MacOS High Sierra (10.13.3) in hopes of creating an Android version of my app, but ran into this error after running `meteor add-platform android`:

```
npm WARN excluding symbolic link node_modules/.bin/nopt -> ../nopt/bin/nopt.js
npm WARN excluding symbolic link node_modules/.bin/semver -> ../semver/bin/semver
npm WARN excluding symbolic link node_modules/.bin/shjs -> ../shelljs/bin/shjs
Subproject Path: CordovaLib                  \
android: added platform
                                              
Your system does not yet seem to fulfill all requirements to build apps for Android.
                     
Please follow the installation instructions in the mobile guide:
http://guide.meteor.com/mobile.html#installing-prerequisites
               
Status of the individual requirements:
✓ Java JDK                                      
✓ Android SDK                
✗ Android target: No android targets (SDKs) installed!
✓ Gradle
```

I've tried the following:
* Adding/removing the android platform multiple times
* Installed Android studio and associated platform versions
* Installed SDK versions from the sdk manager
* Suggestions from #8464, #9746 and a couple of other issues
* brew install android-sdk, brew install java
* Upgraded to Meteor **1.7.0.1**, just for good measure

$ANDROID_HOME is set to `/Users/chip/Library/Android/sdk`, with the following contents:

```
> ls -al $ANDROID_HOME                            
total 8                                                   
drwxr-xr-x  16 chip  staff   512 Jun  5 11:39 .        
drwxr-xr-x   3 chip  staff    96 Feb 13  2017 ..   
-rw-r--r--   1 chip  staff    16 Jun  5 14:22 .knownPackages
drwxr-xr-x   4 chip  staff   128 Feb 13  2017 add-ons         
drwxr-xr-x   8 chip  staff   256 Jun  5 11:43 build-tools
drwxr-xr-x  51 chip  staff  1632 Feb 15  2017 docs      
drwxr-xr-x  17 chip  staff   544 Jun  5 11:38 emulator 
drwxr-xr-x   6 chip  staff   192 Jun  5 11:43 extras       
drwxr-xr-x   3 chip  staff    96 Jun  5 11:39 licenses  
drwxr-xr-x   3 chip  staff    96 Feb 13  2017 patcher            
drwxr-xr-x  19 chip  staff   608 Jun  5 11:38 platform-tools
drwxr-xr-x   7 chip  staff   224 Jun  5 13:25 platforms                                                      
drwxr-xr-x   7 chip  staff   224 Jun  5 13:25 sources
drwxr-xr-x   5 chip  staff   160 Feb 13  2017 system-images
drwxr-xr-x   4 chip  staff   128 Jun  5 11:29 temp
drwxr-xr-x  14 chip  staff   448 Jun  5 11:38 tools
```

And here's what in $ANDROID_HOME/tools:

```
> ls -al $ANDROID_HOME/tools                             
total 2408                                              
drwxr-xr-x  14 chip  staff     448 Jun  5 11:38 .      
drwxr-xr-x  16 chip  staff     512 Jun  5 11:39 ..         
-rw-r--r--   1 chip  staff  818423 Jun  5 11:37 NOTICE.txt
-rwxr-xr-x   1 chip  staff    4853 Jun  5 11:37 android          
drwxr-xr-x  11 chip  staff     352 Jun  5 11:37 bin         
-rwxr-xr-x   1 chip  staff  258856 Jun  5 11:37 emulator                                                     
-rwxr-xr-x   1 chip  staff   95260 Jun  5 11:37 emulator-check
drwxr-xr-x  85 chip  staff    2720 Jun  5 11:37 lib        
-rwxr-xr-x   1 chip  staff   13784 Jun  5 11:37 mksdcard
-rwxr-xr-x   1 chip  staff    1257 Jun  5 11:37 monitor
-rw-r--r--   1 chip  staff   17372 Jun  5 11:38 package.xml                                                  
drwxr-xr-x  12 chip  staff     384 Jun  5 11:37 proguard
-rw-r--r--   1 chip  staff     138 Jun  5 11:37 source.properties
drwxr-xr-x  11 chip  staff     352 Jun  5 11:37 support
```

Any suggestions are greatly appreciated. Thanks.

**Comment by chip** [comment]
Here's a recap of what I did to fix my issue (for MacOS HIgh Sierra 10.13.3):

* Review docs to follow how Meteor integrates with Cordova: 
https://github.com/meteor/meteor/blob/948ec2b05f14559d51548fa536232404943dd512/tools/cordova/README.md 

* Found current `cordova-lib` is at version `7.1.0`:
https://github.com/meteor/meteor/blob/79b5ed8e6524ccbd573c1900188090ec13f81fb9/tools/cordova/index.js

* Installed cordova: `npm i -g cordova@7.1.0`

* List cordova requirements using the **debug** flag: `cordova requirements -d`

* Remove old versions of java:
https://gist.github.com/chip/81d4afc398a0c7300480a39e6dd02940

* Remove old cask versions of the Java & SDK (and anything else that is not needed, located under `/usr/local/Caskroom`:
`brew cask rm java8`
`brew cask rm java`
`brew cask rm android-sdk`

* Use Android Studio to install or remove SDK versions

* Set ANDROID_ROOT in .zshrc:
```
export ANDROID_HOME=~/Library/Android/sdk
export ANDROID_ROOT=${ANDROID_HOME}                                                                 
```

* echo $JAVA_HOME
`/Library/Java/JavaVirtualMachines/jdk1.8.0_25.jdk/Contents/Home`

**Comment by twentyfortysix** [comment]
thanks for the detailed report..
I have been hitting the wall for two days...reinstalling all I could up and down.
The: brew cask rm android-sdk
released the "Android target: No android targets (SDKs) installed!" problem for me.

..now I should solve next errors ;]
